nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/tools/PeakSlider.java

```diff
@@ -558,7 +590,6 @@ public void setActivePeaks(List<Peak> peaks) {
                                         });
                                     }
                                 }
-                                chart.drawPeakLists(true);
```

Comment:
@brucejohnson double check this one especially. This seems to be working okay to me but I am not sure how the cluster matching peaks work, so I don't know if this breaks something.